presto-main/src/main/java/com/facebook/presto/operator/aggregation/GenericAccumulatorFactory.java

```diff
@@ -503,6 +517,7 @@ public void evaluateFinal(BlockBuilder blockBuilder)
         private final List<Integer> orderByChannels;
         private final List<SortOrder> orderings;
         private final PagesIndex pagesIndex;
+        private final Map<Long, RowBlockBuilder> pageMap = new HashMap<>();
```

Comment:
Essentially, this is our own way to main the `GroupedAccumulatorState`. Highly recommend to take a look at how some implementations of `GroupedAccumulatorState` are used. Especially, how to use `ObjectBigArray` as the hash table.

Reply:
Got it taking a look.